src/Sentry.Unity/SentryUnitySdk.cs

```diff
@@ -49,15 +49,19 @@ private SentryUnitySdk(SentryUnityOptions options)
             }
         }
 
-        unitySdk._dotnetSdk = SentrySdk.Init(options);
+        unitySdk._dotnetSdk = Sentry.SentrySdk.Init(options);
+
+        // We can safely call this during initialization. If the SDK self-initialized we're right on time. If the SDK
+        // was initialized manually, the RuntimeOnLoad attributes already triggered, making this call a no-op.
+        StartupTracingIntegration.StartTracing();
 
         // We can safely call this during initialization. If the SDK self-initialized we're right on time. If the SDK
         // was initialized manually, the RuntimeOnLoad attributes already triggered, making this call a no-op.
         StartupTracingIntegration.StartTracing();
 
         if (options.NativeContextWriter is { } contextWriter)
         {
-            SentrySdk.ConfigureScope((scope) =>
+            Sentry.SentrySdk.CurrentHub.ConfigureScope((scope) =>
```

Comment:
You can add to the top (or via csproj iirc): `using Sntry = Sentry.SentrySdk` and use only `Sntry` throughout the file/project

Reply:
This will lead to `ambiguous references` for the namespace `Sentry.Unity` - between `Sentry` and `SentrySdk`.